src/reynir_correct/errtokenizer.py

```diff
@@ -1576,6 +1576,7 @@ def lookup_unknown_words(
     token_stream: Iterable[CorrectToken],
     only_ci: bool,
     apply_suggestions: bool,
+    sss: bool,
```

Comment:
We normallu use pretty descriptive parameter names, so I suggest renaming this to something more meaningful, even if it is long.

Reply:
Done!